File: Tasks/Python Work/cost_of_driving.py
driving_distance = float(input("Enter the driving distance: "))
fuel_efficiency = float(input("Enter the miles per gallon: "))
price_per_gallon = float(input("Enter the price per gallon: "))

cost_of_trip = price_per_gallon * (driving_distance/fuel_efficiency)

print("The cost of driving is $", cost_of_trip, sep="")

File: Tasks/Java Work/CostOfDriving.java
import java.util.Scanner;
	public class CostOfDriving{
		public static void main(String[] args){
		Scanner input = new Scanner(System.in);

System.out.println("Enter the driving distance:");
double drivingDistance = input.nextDouble();

System.out.println("Enter the miles per gallon:");
double milesPerGallon = input.nextDouble();

System.out.println("Enter the price per gallon:");
double pricePerGallon = input.nextDouble();

double costOfTrip = pricePerGallon * (drivingDistance/milesPerGallon);
System.out.println("The cost of the trip is " + costOfTrip);

}
}
